bool dh_status
float32 target_depth
float32 p_scalar
float32 i_scalar
float32 d_scalar
bool auto_dock
int AD_x_error
int AD_y_error
int AD_proximity
